pkg/http/core/task.go

```diff
@@ -30,10 +30,16 @@ func GetTask(c *gin.Context) {
 		return
 	}
 
-	// If there were no kubernetes resources associated with this task ID,
-	// return the default task.
 	if len(resources) == 0 {
-		c.JSON(http.StatusOK, clouddriver.NewDefaultTask(id))
+		// if strings.Contains(id, "-") {
+		// 	c.JSON(http.StatusOK, clouddriver.NewDefaultTask(id))
+		// } else {
```

Comment:
Is there are reason we're keeping these comments?

Reply:
No, an oversight on my part